src/prop_tree.cpp

```diff
@@ -140,7 +140,8 @@ ptree prop_list(const client::history::list& rows,
 
     for (const auto& row: rows)
     {
-        received = ceiling_add(received, row.value);
+        if (row.output.hash() != null_hash)
+            received = ceiling_add(received, row.value);
 
         if (row.spend.hash() != null_hash)
             spent = ceiling_add(spent, row.value);
```

Comment:
Please explain the error that you are attempting to correct. This looks incorrect, as every row assumes an output that may or may not be spent. If there is no output the result should be a sentinel indicating missing necessary data. The missing data can be the result of a server code error or data corruption, or may be the result of a partially-indexed store. But if there is no output for a given input, suppressing that information is undesired behavior.